Sources/LeafKit/LeafRenderer.swift

```diff
@@ -7,51 +7,17 @@ public struct LeafConfig {
 }
 
 protocol LeafCache {
-    func insert(_ document: ResolvedDocument)
-    func load(name: String) throws -> EventLoopFuture<ResolvedDocument>
+    func insert(_ document: ResolvedDocument, on loop: EventLoop) -> EventLoopFuture<ResolvedDocument>
```

Comment:
When would cache operations need to be async? 🤔

Reply:
I really don't know, but I figured it might be easier to go async to sync than the other way around, I guess..

what do youthink?